sdcm/cluster.py

```diff
@@ -4217,8 +4217,10 @@ def wait_total_space_used_per_node(self, size=None, keyspace='keyspace1'):
     def add_nemesis(self, nemesis, tester_obj):
         for nem in nemesis:
             for _ in range(nem['num_threads']):
-                self.nemesis.append(nem['nemesis'](tester_obj=tester_obj,
-                                                   termination_event=self.nemesis_termination_event))
+                nemesis_obj = nem['nemesis'](tester_obj=tester_obj,
+                                             termination_event=self.nemesis_termination_event)
+                nemesis_obj.nemesis_selector_list = nem['nemesis_selector']
```

Comment:
Why do we set it separately after the init completed and not as part of the `init`? Any problem with it?
I would expect addition of one more parameter call here like following:

```
self.nemesis.append(nem['nemesis'](
    tester_obj=tester_obj,
    termination_event=self.nemesis_termination_event,
    nemesis_selector_list=nem['nemesis_selector'],
)
```

Reply:
Because we didn't call it in Nemesis constructor, and run it only in specific nemesis Sisyphus. But i think __init__ could be changes a  bit to support that

Reply:
So, do you plan to do it in scope of this PR?

Reply:
This is still actual.

Reply:
Fixed